README.md

```diff
@@ -20,8 +20,7 @@ A few key features:
 - Online deployment of new [model versions](docs/model_version_policy.md).
 - Support for AI accelerators including [Intel Movidius Myriad VPUs](https://docs.openvinotoolkit.org/latest/openvino_docs_IE_DG_supported_plugins_VPU.html), 
 [GPU](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_supported_plugins_CL_DNN.html) and [HDDL](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_supported_plugins_HDDL.html). 
-- The server can be enabled both on [Bare Metal Hosts](docs/host.md) or in
-[Docker containers](docs/docker_container.md).
+- The server can be enabled both on [Docker containers](docs/docker_container.md).
```

Comment:
This sentence does not make sense anymore.

The server can be run in [docker container](docs/docker_container.md).

Are we sure we want to dispose those instructions

Reply:
fixed